schemas/comorbidity.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "comorbidity",
+  "description": "The collection of data elements related to a donor's comorbidities",
+  "meta": {
+     "parent": "donor"
+  },
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "foreignKey": "sample_registration.program_id",
+        "displayName": "Program ID"
+      },
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "foreignKey": "sample_registration.submitter_donor_id",
+        "displayName": "Submitter Donor ID"
+      },
+      "restrictions": {
+        "required": true,
+        "regex": "#/regex/submitter_id"
+      }
+    },
+    {
+      "name": "prior_malignancy",
+      "description": "Prior malignancy affecting donor.",
+      "restrictions": {
+        "codeList": "#/list/yes_no"
+      },
+      "valueType": "string",
+      "meta": {
+        "displayName": "Prior Malignancy"
+      }
+    },
+    {
+      "name": "laterality_of_prior_malignancy",
+      "description": "If donor has history of prior malignancy, indicate laterality of previous diagnosis. (Reference: caDSR CDE ID 4122391)",
+      "valueType": "string",
+      "restrictions": {
+        "codeList": ["Bilateral", "Left", "Midline", "Not applicable", "Right", "Unilateral, side not specified", "Unknown"],
```

Comment:
Should this be `Side not specified` (capital on first)?

Reply:
also hould list be A-Z sorted?

Reply:
Updated to make first letter capitalized. I believe the list is sorted already.